cmd/spec/path.yaml

```diff
@@ -69,6 +71,8 @@ paths:
   /helptopics/{name}:
     get:
       summary: Return a help topics set by topic name
+      parameters:
+      - $ref: '#/components/schemas/queryParams/Name'
```

Comment:
You can't use the name type `Name`. That one is for query params. This one should be similar to the ID. It has to be in path and do not have any of the additional properties like explode, etc. It is also required as the ID

Reply:
Fixed